libraries/streaming-e2e-tests/build/run-tests.yml

```diff
@@ -0,0 +1,15 @@
+jobs:
+  - job: Install_Drivers_And_Run_Tests
+    variables:
+      ChromeDriverDownloadUrl: 'https://chromedriver.storage.googleapis.com/84.0.4147.30/chromedriver_win32.zip'
```

Comment:
Think about how to sync the version with Microsoft-hosted agent.

https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md